b2sdk/sync/exception.py

```diff
@@ -35,3 +34,25 @@ def __str__(self):
             self.filename,
             self.encoding,
         )
+
+
+class InvalidArgument(B2Error):
```

Comment:
I think that having a single `InvalidArgument` exception is fine.  Having one for each case seems like a lot of trouble, without much benefit.

Reply:
We can always extend it later on because `except` "uses" `isinstance` which works well with child classes...

But I shall add information about this to versioning policy so that public users of v1 know they must expect a more detailed exception - this will let us reduce the amount of proxies if we ever decide to make any changes here.